README.md

```diff
@@ -0,0 +1,10 @@
+# Movies API # 
+A simple API, which stores information about movies.
```

Comment:
Should read like: "...which provides CRUD operations over movies model" or something like this